### PR TITLE
Adding 'thorntail' (previously known as 'wildfly') to 'getting-started'

### PR DESCRIPTION
**Pull request by ibuziuk**
https://github.com/redhat-developer/rh-che/issues/1451

### CODE REVIEW

File: getting-started/thorntail/devfile.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        workdir: ${CHE_PROJECTS_ROOT}/thorntail-rest-http
+  -
+    name: build and run
+    actions:
+      -
+        type: exec
+        component: maven
+        command: "mvn clean install -DskipTests=true && java -jar target/*-thorntail.jar"
+        workdir: ${CHE_PROJECTS_ROOT}/thorntail-rest-http
+  -
```
Comment (amisevsk) [inline getting-started/thorntail/devfile.yaml:62]:
```suggestion
  -
    name: build and run (debugging enabled)
    actions:
      -
        type: exec
        component: maven
        command: |
          mvn clean install -DskipTests=true && \
          java -jar \
          -Xdebug -Xrunjdwp:transport=dt_socket,server=y,address=5005 \
          target/*-thorntail.jar
        workdir: ${CHE_PROJECTS_ROOT}/thorntail-rest-http
  -
```

File: getting-started/thorntail/devfile.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    name: stop
+    actions:
+      -
+        type: exec
+        component: maven
+        command: "echo 'Stopping the application...' && kill $(pidof java) && echo 'The application has been successfully stopped'"
```
Comment (amisevsk) [inline getting-started/thorntail/devfile.yaml:80]:
```suggestion
        command: "echo 'Stopping the application...' && kill $(pidof java) && echo 'The application has been successfully stopped'"
  - 
    name: Debug remote java application
    actions:
      - type: vscode-launch
        referenceContent: |
          {
          "version": "0.2.0",
          "configurations": [
            {
              "type": "java",
              "name": "Debug (Attach) - Remote",
              "request": "attach",
              "hostName": "localhost",
              "port": 5005
            }]
          }
```

**Review by amisevsk — APPROVED** [review]
Tested on prod-preview and it works without issue. 

One suggestion: add debug configuration as in https://github.com/eclipse/che-devfile-registry/pull/66. I've added suggestions below for what those changes would look like.

**Comment by ibuziuk** [comment]
@amisevsk thanks for the great improvements. applied!

**Pull request merged**